experimental/structs.md

```diff
@@ -0,0 +1,57 @@
+# [JavaScript Structs: Fixed Layout Objects and Some Synchronization Primitives](proposal-structs)
+
+# Structs
+
+```js
+interface Struct <: Node {
+    shared: boolean;
+    id: Identifier | null;
```

Comment:
As far as I can tell from the proposal, `struct` must always have a name so `| null` is not valid here.

Reply:
Basically, there should be just `StructDeclaration` and you can probably reuse `<: ClassDeclaration` for it as a base.